pontoon/base/static/css/style.css

```diff
@@ -187,18 +187,30 @@ th {
   position: absolute;
 }
 
-.table-sort th.asc i:after {
+.table-sort th.sort-order i:after {
   content: "";
   display: inline-block;
   margin-top: 5px;
 }
 
-.table-sort th.desc i:after {
+.table-sort th.rev-order i:after {
   content: "";
   display: inline-block;
   margin-top: -5px;
 }
 
+.table-sort th.priority.sort-order i:after {
+  content: "";
+  display: inline-block;
+  margin-top: -5px;
+}
+
+.table-sort th.priority.rev-order i:after {
+  content: "";
+  display: inline-block;
+  margin-top: 5px;
+}
```

Comment:
I guess you `display` and `margin-top` don't need to be repeated.

And you also need to add a selector for the `.unreviewed-status`.

Reply:
If I remove those then the alignment for the arrow is off. 

So, I opted to add a new class (`desc-sort`) to those columns that need to be in descending order as default, and then I added  a selector for that class. I think this will make it more future proof too.